static/app.webmanifest

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "Zulip (Web)",
+    "short_name": "Zulip",
+    "description": "Organized chat for distributed teams.",
+    "start_url": "/",
+    "display": "standalone",
+    "icons": [
+        {
+            "src": "/static/images/logo/zulip-icon-192x192.png",
+            "sizes": "192x192",
+            "type": "image/png"
+        },
+        {
+            "src": "/static/images/logo/zulip-icon-512x512.png",
+            "sizes": "512x512",
+            "type": "image/png"
+        }
+    ],
+    "screenshots": [
+        {
+            "src": "/static/images/landing-page/hello/generated/screen-1-mobile-2x.webp",
+            "sizes": "786x1436",
+            "type": "image/webp",
+            "form_factor": "narrow",
+            "label": "Screenshot showing the Zulip web app user interface."
+        },
+        {
+            "src": "/static/images/landing-page/hello/generated/screen-1-2x.webp",
+            "sizes": "2190x992",
+            "type": "image/webp",
+            "form_factor": "wide",
+            "label": "Screenshot showing the Zulip web app user interface."
+        }
+    ]
+ }
```

Comment:
We generally aim to have newlines at end of files; is not having one important?

Reply:
No, I don't think so that's the case, I'll add one.